wg-lts/charter.md

```diff
@@ -0,0 +1,88 @@
+# WG LTS Charter
+This charter is a WIP.
+
+This charter adheres to the [wg-governance] guidance, as well as
+adheres to the general conventions described in the [Kubernetes
+Charter README] and the Roles and Organization Management outlined
+in [sig-governance], where applicable to a Working Group.
+
+## Scope
+The Long Term Support Working Group (WG LTS) is organized with the
+goal of provide a cross SIG location for focused collection of
+stakeholder feedback regarding the support stance of Kubernetes
+project in order to inform proposals for support improvements.
+
+"WG LTS" is simply shorter than "WG To LTS Or Not To LTS" or "WG
+What Are We Releasing And Why And How Is It Best Integrated, Validate,
+And Supported", but should NOT be read in that shortness to imply
+establishing a traditional LTS scheme is the foregone conclusion of
+the WG.
+
+### In Scope
+* What is a Kubernetes release?
+* What is a supported release?  Ie: Stable branches to which critical fixes are backported.
+ * Number of community supported branches.
+ * Duration of community support per supported branch.
+ * Upgrade path considerations.
+* Costs of Kubernetes releases in terms of:
+ * Infrastructure
+ * People
+* Process: how/where code changes are backported to community supported branches.
+
+### Out of Scope
+* The lifecycle of projects outside of the Kubernetes org.
+* Technical and end-user support:  The WG will make recommendations
+  around support to those responsible for relevant code and responsible
+  for the release engineering operations and automation, but does not
+  own code itself.
+* Code, test case, automation implementations:  this is a working
+  group, no code implementation is the responsibility of this Working
+  Group.
+
+## Stakeholders
+* Kubernetes end users
+* Kubernetes cluster operators
+* Kubernetes vendors, distributions, and hosting providers
+* SIG Release
+* SIG Cluster Lifecycle
+* SIG Cloud Provider
+* SIG Testing
+* SIG Architecture & Steering Committee
+* ...arguably all SIGs in as much as proposals may impact the way the community developers, ships and maintains its code
```

Comment:
SIGs API Machinery, CLI, and Node would be affected, in particular, if supported version skew were to change.

Reply:
I can easily add these, but also wonder:  In the sense of this being a WG not a SIG and thus not owning code, and KEPs being the form of any possible eventual change proposals...how specific should we be in this document?  As I start adding, I can imagine additional arguments for why basically every SIG is a stakeholder, and even a blocking stakeholder in some cases.  But somehow I think we need to differentiate between inputs / stakeholder involvement and outputs / blocking stakeholders.

Per-KEP I can imagine the possibility of some being more targeted and shorter stakeholder list, and some that would truly be all SIGs are blocking stakeholders on approval.  I'd like to pragmatically allow that, and here in the document be indicating the intent to be as maximally inclusive as possible, without having to list every SIG and keep the list up to date as the SIG list morphs over the coming year.

If any wordsmiths have a solid proposal on how to do that I'd love suggestions.

Reply:
Many of the stakeholders listed are not blocking stakeholders.

If you plan to seriously investigate the issue of version skew, then the SIGs I mentioned need to be consulted. I think they're far more significant than, say, SIG cloud provider. You can omit them from the doc, but that wouldn't give me confidence in the outcome of the WG.

Reply:
cc @jagosan

Reply:
Added SIGs API Machinery, CLI, and Node.

Also revamped this stakeholders section based on feedback from folks in the first WG LTS meeting (https://youtu.be/i1sCgfLFU-I?list=PL69nYSiGNLP13_zDqYfUjfLZ2Lu9a3pv-).